test/MUXControlsTestApp/App.xaml.cs

```diff
@@ -201,8 +201,13 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
         {
             _isRootCreated = false;
             // Load the resource dictionary now
-            Application.LoadComponent(AdditionStylesXaml, new Uri(
-                            "ms-appx:///Themes/AdditionalStyles.xaml"), ComponentResourceLocation.Nested);
+            // Since the resource is only availble with ScrollViewer feature enabled, try this but expect it to fail sometimes
+            try
+            {
+                Application.LoadComponent(AdditionStylesXaml, new Uri(
+                                "ms-appx:///Themes/AdditionalStyles.xaml"), ComponentResourceLocation.Nested);
+            }
+            catch { }
```

Comment:
What in this change required this ? How was this working before this change ?

Reply:
The try catch is necessary. Before that change, the app crashed if we did not have the ScrollerViewer test UI project as the file wouldn't exist and we couldn't load it.

Reply:
I see. It was always there in MuxControls.sln so we never hit this case. For every combination that does not include ScrollViewer this will throw an exception and we will swallow it, which makes is not an 'exception' case. It can be quite annoying when debugging with exceptions enabled.  Can we add a conditional compile piece here depending on ScrollViewer feature being enabled instead ?

Reply:
Are there conditional compiles for the feature areas? I know there's one for innerloop however, if you work on ScrollViewer, that would not load the file none the less. 

One way to handle this would be to check if the file exists, however that would introduce asynchronity in here.

Reply:
I see that you will add a conditional for scroller with #2192 . Should I wait until it goes through and merge then or add that here too and later resolve possible merge conflicts?

Reply:
Weird. Those changes are unrelated, should likely go away with a merge from master. you can ignore those.